tests/jballerina-unit-test/src/test/resources/test-src/jvm/largeMethods2/modules/records/query-large-record-array.bal

```diff
@@ -76,19 +76,15 @@ function testQueryExpWithinSelectClause() {
     {}, {},{}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {},{}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {},{}, {},
     {}, {}, {}, {}, {}, {}, {}, {}, {}, {},{}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {},{}, {}, {}, {}, {}, {},
     {}, {}, {}, {}, {}, {},{}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {},{}, {}, {}, {}, {}, {}, {}, {}, {}, {},
-    {}, {},{}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {},{}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {}, {},{}, {},
```

Comment:
Shall we create a task issue revert these changes, after merging https://github.com/ballerina-platform/ballerina-lang/pull/41017 changes. We have commented some lines below instead of removing

Reply:
Maybe if we merge that first, we may not need to change these

Reply:
Shall we revert this change